moped-editor/src/components/Maps/ComponentsDrawControl.js

```diff
@@ -123,15 +119,14 @@ const ComponentsDrawControl = React.forwardRef(
       position: "top-right",
       displayControlsDefault: false, // Disable to allow us to set which controls to show
       default_mode: "simple_select",
-      clickBuffer: activeLineWidth,
+      clickBuffer: 8, // pixel width of click tolerance to make a point or line active
```

Comment:
Some of the line segment styles in the map were set statically and some of them were set dynamically based on the zoom level of the map. They are all dynamic now so `activeLineWidth` no longer exists. This buffer must be a static pixel value so we will have to adjust it here in the future.

Reply:
I noticed this in this PR when reviewing. This is a billion times better -- thank you for finding this!